internal/scheduler/metrics/cycle_metrics.go

```diff
@@ -364,6 +388,24 @@ func (m *cycleMetrics) ReportSchedulerResult(result scheduling.SchedulerResult)
 				nodePreemptiblityStats.Reason).Set(1.0)
 		}
 
+		nodes, err := schedulingStats.NodeDb.GetNodes()
+		if err != nil {
+			log.Errorf("unable to generate node stats as failed to get nodes from nodeDb %s", err)
+		} else {
+			for _, node := range nodes {
+				isSchedulable := strconv.FormatBool(!node.IsUnschedulable())
+				for _, resource := range node.GetAllocatableResources().GetResources() {
+					currentCycle.nodeAllocatableResource.WithLabelValues(node.GetPool(), node.GetName(), node.GetExecutor(), node.GetReportingNodeType(), resource.Name, isSchedulable).Set(float64(resource.RawValue))
```

Comment:
`RawValue` needs to be scaled.

You can call `QuantityAsFloat64` defined [here](https://github.com/armadaproject/armada/blob/27cfc6ed1639ef6cdca266e2f5f6353176b7fb00/internal/common/resource/resource.go#L48) on resource.Value